src/pages/SmartWeaveExplorer/Contract/Contract.vue

```diff
@@ -230,8 +231,13 @@
                   </b-form-radio-group>
                 </b-col>
 
-                <b-button class="btn btn-refresh rounded-pill mb-3 mb-sm-0" @click="refreshData"
-                  >Refresh data
+                <b-button
+                  class="btn btn-refresh d-flex justify-content-center align-items-center rounded-pill mb-3 mb-sm-0"
+                  @click="refreshData"
+                  ><p class="m-0" v-if="interactionsLoaded">Refresh data</p>
```

Comment:
I would add here a filter: Transaction type: contracts/interactions (similair radio options can be found above other lists)

Reply:
Done at frontend side, but needs to be done at backend. Currently waiting for '&type=' parameter in creator end point